install/nginxproxymanager-install.sh

```diff
@@ -141,9 +141,9 @@ msg_ok "Set up Enviroment"
 
 msg_info "Building Frontend"
 cd ./frontend
-$STD pnpm install
-$STD pnpm upgrade
-$STD pnpm run build
+$STD yarn cache clean --silent --force 
```

Comment:
Have you tested this? Where is yarn installed? pnpm install section is not removed? https://github.com/tteck/Proxmox/commit/edf3803362cb7ad152fb386997def01edf5ef57e#diff-e676f31f25717f8123cc0904ec5973dee3dc53b86e2b0ff7bec0eeb72c6ce82fR132

Reply:
Yep, you can run the bash script against the develop Branch

Reply:
https://github.com/NginxProxyManager/nginx-proxy-manager/blob/b3de76c945b087b9a585faff128ca655170f8ea1/docker/Dockerfile#L44-L47

Reply:
✓ Stopped Services
 ✓ Cleaned Old Files
 ✓ Downloaded NPM v2.12.1
 ✓ Setup Enviroment
 / Building Frontend   
[ERROR] in line 143: exit code 0: while executing command yarn cache clean --silent --force &> /dev/null

running against the dev branch ...

Reply:
Lol. Work on my systems fine :D fck NPM

Reply:
> https://github.com/NginxProxyManager/nginx-proxy-manager/blob/b3de76c945b087b9a585faff128ca655170f8ea1/docker/Dockerfile#L44-L47

That is backend install, not the same as frontend install which you are changing in this PR